### ISSUE TITLE
User should be able to View Worksite accessibility

### ISSUE THREAD

**Issue by ayoutzy** [ayoutzy/Its_a_GO#5]

**Comment by ayoutzy** [comment]
Cleared.html and stayhome.html were created
